cli/jobs/pipelines/automl/text-classification-multilabel-paper-categorization-pipeline.yml

```diff
@@ -0,0 +1,42 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineJob.schema.json
+type: pipeline
+
+description: Pipeline using AutoML Text Classification Multilabel task
+
+display_name: pipeline-with-text-classification-multilabel
+experiment_name: pipeline-with-automl
+
+settings:
+  default_compute: azureml:gpu-cluster
+
+inputs:
+  text_classification_multilabel_training_data:
+    type: mltable
+
+    path: ../../automl-standalone-jobs/cli-automl-text-classification-multilabel-paper-cat/training-mltable-folder
+  text_classification_multilabel_validation_data:
+    type: mltable
+    path: ../../automl-standalone-jobs/cli-automl-text-classification-multilabel-paper-cat/validation-mltable-folder
+
+jobs:
+  text_classification_multilabel_node:
+    type: automl
+    task: text_classification_multilabel
+    log_verbosity: info
+    primary_metric: accuracy
+    limits:
+      max_trials: 1
+      timeout_minutes: 60
+    target_column_name: terms
+    training_data: ${{parent.inputs.text_classification_multilabel_training_data}}
+    validation_data: ${{parent.inputs.text_classification_multilabel_validation_data}}
+    # currently need to specify outputs "mlflow_model" explicitly to reference it in following nodes
+    outputs:
+      best_model:
+        type: mlflow_model
+  show_output:
```

Comment:
what's typical user next step when thy have an automl outputed model?
Maybe we can make the example not so dummy